src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskAttachmentEntity.kt

```diff
@@ -0,0 +1,19 @@
+package fi.metatavu.lipsanen.projects.milestones.tasks
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table(name = "task_attachment")
+class TaskAttachmentEntity {
+
+    @Id
+    lateinit var id: UUID
+
+    @ManyToOne
+    @JoinColumn(name = "task_id", nullable = false)
+    lateinit var taskId: UUID
```

Comment:
This might be not needed at all since you manage the relation from the parent's (TaskEntity side)